source/main/scripting/CBytecodeStream.cpp

```diff
@@ -35,18 +35,20 @@ CBytecodeStream::~CBytecodeStream()
         fclose(f);
 }
 
-void CBytecodeStream::Write(const void* ptr, AngelScript::asUINT size)
+int CBytecodeStream::Write(const void* ptr, AngelScript::asUINT size)

```

Comment:
This change is not associated with AngelScript update, it's change of our own code and should have been in separate commit. 

Also, why return int when the indicated states are just "success/failure"?

Reply:
From the changelog: ***Both Read and Write in asIBinaryStream can now return an error code to signal failure***
http://www.angelcode.com/angelscript/sdk/docs/manual/classas_i_binary_stream.html

So this change is required otherwise the build fails
```
error C2555: 'CBytecodeStream::Read': overriding virtual function return type differs and is not covariant from 'AngelScript::asIBinaryStream::Read'
```

Reply:
You're right, I just ran into it while building.